docs/exit_validation.md

```diff
@@ -111,12 +111,12 @@ Absence of challenges within some period (like `sla_margin`) must result in clie
 
 ## In-flight exits
 
-With MoreVP, we need to handle another type of exit game, which is the in-flight exit game, as specced out [here](morevp.md).
+With MoreVP, we need to handle another type of exit game, which is the in-flight exit game, as specced out [here](docs/morevp.md).
```

Comment:
the link is not working

Reply:
yeah, I always forget how does GH render these. Fixed throughout all `md` files. Thx!